### PR TITLE
Updating settings-local.js code

### PR DESCRIPTION
**Pull request by magicmicah**
Based off this issue, setting the settings.js configuration to the new setting helped us in our email notifications. Deck UI was broken for us until we applied this change. 

https://github.com/spinnaker/spinnaker/issues/2484#issuecomment-418443115

### CODE REVIEW

**Review by magicmicah — COMMENTED** [review]
Testing with just settings.js shows the Loading failure. settings-local.js is better.

**Review by dorbin — APPROVED** [review]
LGTM
Will let @ezimanyi take a look.

File: setup/features/notifications/index.md [hunk 1]
```diff
@@ -26,13 +26,15 @@ See also [`hal config notifications`](/reference/halyard/commands/#hal-config-no
...
-Notification configurations are in echo.yml and settings.js. For changes to echo.yml, create echo-local.yml and put your changes in there. You can customize settings.js directly.
+Notification configurations are in echo.yml and settings-local.js. For changes to echo.yml, create echo-local.yml and put your changes in there. You can customize settings-local.js directly.
 
 ## Where to put echo-local.yml and settings.js
 
-If you use Halyard to configure Spinnaker, put echo-local.yml and settings.js in `~/.hal/{deployment}/profiles/`.
+If you use Halyard to configure Spinnaker, put echo-local.yml  `~/.hal/{deployment}/profiles/`. 
 
-If you don’t use Halyard, put echo-local.yml in the same place as the current echo.yml, in `/opt/spinnaker/config`, and put settings.js in `/opt/deck/html/`.
+For settings-local.yml, follow the [Custom Profile for Deck](/reference/halyard/custom/#custom-profile-for-deck) reference and place it in the following location `~/.hal/{deployment}/profiles/settings-local.js`.
```
Comment (ezimanyi) [inline setup/features/notifications/index.md]:
```suggestion
For settings-local.js, follow the [Custom Profile for Deck](/reference/halyard/custom/#custom-profile-for-deck) reference and place it in the following location `~/.hal/{deployment}/profiles/settings-local.js`.
```

**Review by ezimanyi — APPROVED** [review]
Looks good, just one suggestion!

**Pull request merged**